core/models.py

```diff
@@ -156,14 +156,17 @@ class Entry(BaseModel):
     original_entry_id = models.PositiveIntegerField(unique=True)
     lead = models.ForeignKey(Lead, on_delete=models.CASCADE)
     original_lang = models.CharField(max_length=50)
-    excerpt_en = models.TextField()
+    excerpt = models.TextField()
```

Comment:
are we only pulling english excerpts ??

Reply:
In fact, we are pulling form the `excerpt` column of `entry` table. Whatever is there gets pulled without language information. Do you think we should store language info/translate accordingly?

Reply:
I think it is okay to pull the excerpts irrespective of the language. The model is multi-lingual and able to generate predictions and embeddings (atleast for lang: en, fr, es, ar, portuguese). For other lang, preds and embeds might not be that good.

Reply:
Sounds good.